integrations/astra/src/haystack_integrations/document_stores/astra/astra_client.py

```diff
@@ -5,6 +5,10 @@
 
 from astrapy.api import APIRequestError
 from astrapy.db import AstraDB
+
+# For version tracking
+from haystack import __name__ as integration_name
```

Comment:
I would just hardcode the string `haystack` at the module level and save an import call, something like
```py
CALLER_NAME = 'haystack'